subscription.go

```diff
@@ -2,19 +2,36 @@ package pubsub
 
 import (
 	"context"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+type EventType uint8
+
+const (
+	UNKNOWN EventType = iota
```

Comment:
this is unnecessary.

Reply:
My understanding is that this is best practice so that an empty `EventType` doesn't have a valid value by accident. Is that incorrect/incomplete?

Reply:
it's not, it's just redundant.